mirrord/cli/src/config.rs

```diff
@@ -396,12 +396,22 @@ pub(super) struct PortForwardArgs {
     /// If the remote is given as an ip, this is parsed as soon as mirrord starts.
     /// Otherwise, the remote is assumed to be a hostname and lookup is performed in the cluster
     /// after a connection is made to the target.
+    /// Multiple forwarding mappings are each passed with -L.
     #[arg(short = 'L', long)]
-    pub port_mappings: Vec<PortMapping>,
+    pub port_mappings: Vec<AddrPortMapping>,
```

Comment:
I still think this should be called `port_mapping`, singular, BTW.
A mapping maps multiple keys two multiple values.
And even if a mapping would only mean one key->value item, I would still find it nicer to write `... --port-mapping 80:3000 --port-mapping 8080:3030` than `... --port-mappings 80:3000 --port-mappings 8080:3030`.
We could ask for more feedback from the team if you're not convinced :)

Reply:
We can leave `--port-mappings` as an [`alias`](https://docs.rs/clap/latest/clap/struct.Arg.html#method.alias) for backwards compatibility.

Reply:
i agree, the plural is misleading